.github/workflows/rust-clippy.yml

```diff
@@ -0,0 +1,51 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
```

Comment:
Why is this sentence here? Depending on the reason, you might want the same sentence in `.github/workflows/ci.yml`.

Reply:
This is not our script. We should leave it as is in so in the future we get a fresh new one instead of fixing it.

Reply:
This is just a _caveat emptor_ that you're using third-party code on GH Actions and if something breaks it's not GHs fault. No harm in keeping it or removing it either way, but might make it easier to track down the provenance.

Reply:
> This is not our script.

Let's add a link to the source. Also, what's the license of the source?

Reply:
Idk, it's from the Github action item, and I just click on it and make minor modification. 
I'm pretty sure it's open license, because there are plenty projects use this, although I don't have time to find the exact original license of this source.

Reply:
> I don't have time to find the exact original license of this source.

That sounds like a blocker.